dev/com.ibm.ws.security.javaeesec_fat/fat/src/com/ibm/ws/security/javaeesec/fat_helper/JavaEESecTestBase.java

```diff
@@ -4,7 +4,7 @@
  * are made available under the terms of the Eclipse Public License 2.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-2.0/
- * 
+ *
```

Comment:
```suggestion
 * 
```

We should remove this file from the commit, so that the history (if anyone ever looks at it) for the file, doesn't have a blank change

Reply:
File has been removed from commit.